src/server/server.zig

```diff
@@ -18,6 +18,18 @@ pub const storage = @import("storage.zig");
 
 const command = @import("command/_command.zig");
 
+pub const UserCommandData = struct {
```

Comment:
Wouldn't it be better to call it `WorldEditData`?

Reply:
I wanted to prepare a place for future command state if it ever exists, it doesn't really make sense to have every bit of command state separately IMO, a lot of potential additional init and deinit calls would be required.

Reply:
Well, in my opinion we shouldn't sacrifice readability for an uncertain future.
And if we do discover that many commands need user state, then we should probably solve this in a more generic way, such that mods (if they'll ever be a thing) can also use it.

Reply:
As you prefer, although I don't think it is significantly harming readability - it is not a lie, just the name is a umbrela term.

> And if we do discover that many commands need user state, then we should probably solve this in a more generic way, such that mods (if they'll ever be a thing) can also use it.

Definitely outside of the scope of your PR.